static/js/index.js

```diff
@@ -5741,11 +5724,8 @@ function getCpAtLevel(id, form, level, isMax) {
     }
     let pkmn = [];
     if (cpMultipliers[level]) {
-        if (form === 0 || typeof masterfile.pokemon[id].forms[form].attack === 'undefined') {
-            pkmn = masterfile.pokemon[id];
-        } else {
-            pkmn = masterfile.pokemon[id].forms[form];
-        }
+        form === 0 || Object.keys(masterfile.pokemon[id].forms[form].attack).length === 0 ?
+            pkmn = masterfile.pokemon[id] : pkmn = masterfile.pokemon[id].forms[form];
```

Comment:
I am not sure about operator priorities here. I recommend `pkmn = <blah> ? <stuff> : <other stuff>` instead.

Reply:
I agree, setting the variable based on the null coalescing operator would be much cleaner/better.
```
pkmn = form === 0 || Object.keys(masterfile.pokemon[id].forms[form].attack).length === 0
    ? masterfile.pokemon[id]
    : masterfile.pokemon[id].forms[form];
```